packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.test.tsx

```diff
@@ -106,4 +111,11 @@ const defaultStripeOverride: Pick<
       error: undefined,
     };
   },
+  confirmCardSetup: async () => {
+    await wait(500);
```

Comment:
I used a delay in stories to give a change to simulate a loading spinner, but we don't need it in tests.